### PR TITLE
Fix: Centralize carrier time durations into an internal package everything else imports.

### LINKED ISSUE #195: Carrier timeouts should be configurable
We have various timeouts with arbitrary values (e.g. https://github.com/SUSE/carrier/blob/main/paas/install_client.go#L25). In some cases (e.g. acceptance tests) we may need higher timeouts and it would be desirable to be able to multiply the default value by a factor using an environment variable.

## Acceptance Criteria

- The user should be able to multiple the default values for all timeouts by a factor using an environment variable. This could be on a per context basis too (e.g. network, cpu etc)

We will probably need to extract all timeouts to a separate package and import it when a timeout value is needed. To be able to use a multiplier, we will access the timeout values through getter methods that will make sure the multiplier is applied when available.

### PR DESCRIPTION
**Pull request by andreas-kupries**
Fixes #195 .

### CODE REVIEW

File: internal/services/catalog_service.go [hunk 1]
```diff
@@ -280,7 +280,8 @@ func (s *CatalogService) CreateBinding(bindingName, org, serviceName string) (in
 // to an Application.
 func (s *CatalogService) GetBindingSecret(bindingName string) (*corev1.Secret, error) {
 	// TODO: Replace hardcoded timeout with a constant
```
Comment (jimmykarily) [inline internal/services/catalog_service.go:282]: remove TODO?

File: internal/duration/duration.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
+// Package duration defines the various durations used throughout
+// Carrier, as timeouts, and other.
+package duration
+
+import (
+	"time"
+)
+
+const (
+	multiplier    = 1
```
Comment (jimmykarily) [inline internal/duration/duration.go:10]: I don't see a way to change the multiplier are run time. Maybe we can create a `Multiplier()` method that return either the value of `CARRIER_TIMEOUT_MULTIPLIER` environment variable or `1` if the variable is not set. What do you think?

Comment (andreas-kupries) [reply internal/duration/duration.go:10]: Heh. I had defered that yesterday evening to today. The commit is currently waiting to be made (just a bit of testing).
The variable in the commit is `CARRIER_DURATION_MULTIPLIER` ... Hm. Timeout fits better, all the affected durations are timeouts (poll interval and user abort are not affected, and not timeouts per se). ... Will change.

File: internal/services/catalog_service.go (hunk 1 above)
Comment (andreas-kupries) [reply internal/services/catalog_service.go:282]: yes.

**Review by jimmykarily — APPROVED** [review]

**Pull request merged**